### DISCUSSION TITLE
64 Bit Raspberry PI version

### DISCUSSION THREAD

**Discussion by tcoombs** [rustdesk/rustdesk#1172] [General]
I would like to request a future enhancement for the 64 bit version for Raspberry PI OS. I've had to downgrade my Raspberry PI 4 appliances to 32 bit to get Rustdesk working (love the product by the way..)

**Comment by rustdesk** [comment]
will do next release, because Sciter sdk does not work well on arm64, we are rewriting with flutter.

**Reply by tce-darshan** [reply]
Thank you very much for the awesome product. Any update on 64 bit version?

**Comment by tcoombs** [comment]
Thanks Guys!

**Comment by Nurgiel** [comment]
Looks like it's still not fixed 3 years later. I'm getting:

Depends: libxdo3 but it is not installable
Recommends: libayatana-appindicator3-1 but it is not installable

on Raspberry Pi 5 aarch64 Debian 12 bookworm

I don't want to downgrade to 32bit, I think I should just install Debian 10 instead of default "Linux raspberrypi 6.12.20+rpt-rpi-2712 Debian 12 bookworm aarch64" that arrived with this little PC.
